index.html

```diff
@@ -30,7 +30,7 @@
       }
     };
     globalThis.packageVersion = "23.09.0-beta.1";
-    globalThis.buildVersion = "230906.190949";
+    globalThis.buildVersion = "230907.120939";
```

Comment:
Is there any reason to include a buildVersion update?

Reply:
Urgh, must be automatically generated when running `make compile` and forgot to remove it.